scripts/globals/mobskills/Colossal_Blow.lua

```diff
@@ -11,21 +11,30 @@ require("scripts/globals/monstertpmoves");
 ---------------------------------------------------
 
 function onMobSkillCheck(target,mob,skill)
- local mobID = mob:getID();   --(16908294 ,16908301 ,16908308 =omega ,16933124=proto-omega)
- local mobhp = mob:getHPP();
-   if (mobID == 16933124 and mobhp < 30 ) then -- omega second quadripedform
-	  return 0;
-   end
-   return 1;
+    local currentForm = mob:getLocalVar("form") -- this var is only set for proto-omega
+
+    if (currentForm == 2) then
+        return 0;
+    end
+    return 1;
 end;
 
 function onMobWeaponSkill(target, mob, skill)
 
-	local numhits = 1;
-	local accmod = 0.8;
-	local dmgmod = 5;
-	local info = MobPhysicalMove(mob,target,skill,numhits,accmod,dmgmod,TP_NO_EFFECT);
-	local dmg = MobFinalAdjustments(info.dmg,mob,skill,target,MOBSKILL_PHYSICAL,MOBPARAM_SLASH,info.hitslanded);
-	target:delHP(dmg);
-	return dmg;
-end;
+    local currentHP = target:getHP();
+    -- remove all by 5%
+    local damage = 0;
+
+    -- if have more hp then 30%, then reduce to 5%
+    if (currentHP / target:getMaxHP() > 0.2) then

```

Comment:
That was a copy and paste from Throat Stab since the old version of the script simply had it doing damage. I'll correct that in a moment.